### DISCUSSION TITLE
Imdb description when hovering on a video title

### DISCUSSION THREAD

**Discussion by stup52** [linuxmint/discussions#707] [Ideas]
I guess we all backup our videos as copies or mp4 files and when there are many it is sometimes difficult to remember what the dvd film was about.
When I left the Dark Side (Windows) it had an option to select the behaviour of folders - music, pictures, videos, not the same as linux designated folder types. If linux mint could do similar with a folder such that then when hovering on a copied/backed up dvd in a dvd or mp4 designated folder the Imdb description popped up for the film it might be a useful addition to some, well, me at least.

Hope that made sense...

**Comment by Jopp-gh** [comment]
Absolutely, it makes sense. 

The issue I see is two fold:
Mint is privacy focused and such features wouldn't work without internet. This feature sounds kind of telemetry stuff; I personally wouldn't want such a background data transfer.  

I suggest, to place your movies in top folders describing their genre.
